registry.k8s.io/images/k8s-staging-sig-storage/images.yaml

```diff
@@ -294,6 +295,8 @@
     "sha256:4e97e6f8c122c87253c89fce466e760f88122aa4a7b21677fad4c603144cc0dd": [ "v1.14.0" ]
     "sha256:76ead9378a4b92c4da92a934a234cdd4469dd3628fb51a845b15fca7d2a9979b": [ "v1.15.0" ]
     "sha256:1ec16928aa355e3dafdc84be2acf88d7d9124816e4cd580411536eae064f1d37": [ "v1.16.0" ]
+    "sha256:7d4525de48a1b7e775120ea69828f3fefa0c65346964d7163e4bed16799772c9": [ "v1.17.0" ]
+    "sha256:b4c6306c9f23ca8bb15c90b0c28e2823d7b64d2ced6fa8df9de3cc4c4f0a9d5c": [ "v1.17.0-windows-hp" ]
```

Comment:
Is this added manually?  If so, it's going to be removed when someone else runs the script.

Reply:
yes it's manually added, will modify the script to also support `v1.17.0-windows-hp` tag later

Reply:
I mean in another PR

Reply:
could you approve this PR first? @xing-yang

Reply:
I have created another PR fixing the script: https://github.com/kubernetes/k8s.io/pull/7728